master/docs/developer/database.rst

```diff
@@ -1855,3 +1855,52 @@ id.
 
 If this weakness has a significant performance impact, it would be acceptable to
 conditionalize use of the subquery on the database dialect.
+
+Testing migrations with real databases
+--------------------------------------
+
+By default Buildbot test suite uses SQLite database for testings database
```

Comment:
I checked section above for that style policy --- it doesn't follow it either :)
It's a good policy IMO.